docs/starters.yml

```diff
@@ -1685,6 +1685,21 @@
     - Full setup instructions included
     - Documented to get you set up with Craft CMS quickly
     - Code referenced in repo
+- url: https://gatsby-starter-buttercms.netlify.com/
+  repo: https://github.com/ButterCMS/gatsby-starter-buttercms
```

Comment:
I have a few comments on the starter itself.

You have a Readme.md that says
```
.gitignore: This file tells git which files it should not track / not maintain a version history for.

.prettierrc: This is a configuration file for Prettier. Prettier is a tool to help keep the formatting of your code consistent.
```
but I can't seem to find a gitignore or prettier file, could you add it or remove these lines?

It would also be handy if you add https://gatsby-starter-buttercms.netlify.com/ to your README so people can easily view it from the github page.